### PR TITLE
Typeahead issues 

### LINKED ISSUE #155: Typeahead bugs
Fix typeahead bugs.

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Attempt to fix typeahead bug 
2.  Fix typeahead input styling. 

Closes #155

### CODE REVIEW

File: pokeapp/src/components/Sidebar.js [hunk 1]
```diff
@@ -258,8 +258,8 @@ const SideBar = ({
 
   return (
     <div className="sidebar">
-      {selectedPokemon.types ? sidebarInformation() : <p>Side Bar</p>}
-    </div>
+      {selectedPokemon.types ? sidebarInformation() : null}{" "}
+    </div> // add pokeball here
```
Comment (connielion) [inline pokeapp/src/components/Sidebar.js:262]: nice comment

**Review by connielion — APPROVED** [review]
Looks good!

File: pokeapp/src/components/Sidebar.js [hunk 2]
```diff
@@ -258,8 +258,8 @@ const SideBar = ({
 
   return (
     <div className="sidebar">
-      {selectedPokemon.types ? sidebarInformation() : <p>Side Bar</p>}
-    </div>
+      {selectedPokemon.types ? sidebarInformation() : null}{" "}
```
Comment (Lap343) [inline pokeapp/src/components/Sidebar.js:261]: Do you still need the {""} looks like the same thing Garret had a bit back

**Review by Lap343 — APPROVED** [review]
Other than what I commented it looks good

File: pokeapp/src/components/Sidebar.js (hunk 2 above)
Comment (JayDampitan) [reply pokeapp/src/components/Sidebar.js:261]: Didn't really mean to add that, just appeared after I added the comment, weird.

**Review by BlackCubes — APPROVED** [review]
Looking JOOICY!

**Pull request merged**